tests/dryrun/install_test.go

```diff
@@ -242,3 +245,53 @@ func TestCustomPortsInstallation(t *testing.T) {
 
 	t.Logf("%s: test complete", time.Now().Format(time.RFC3339))
 }
+
+func TestConfigValuesInstallation(t *testing.T) {
+	tmpdir, err := os.MkdirTemp("", "embedded-cluster-test")
+	if err != nil {
+		t.Fatalf("failed to create temp dir: %v", err)
+	}
+	defer os.RemoveAll(tmpdir)
+	tempfile := filepath.Join(tmpdir, "values.yaml")
+	os.WriteFile(tempfile, []byte(`
```

Comment:
let's move this to a yaml file under the `assets` directory and use go embed similar to the other tests

Reply:
done